src/metabase/api/user.clj

```diff
@@ -31,20 +31,21 @@
    [metabase.util.password :as u.password]
    [toucan2.core :as t2]))
 
-(defsetting user-visibility
-  (deferred-tru "Note: Sandboxed users will never see suggestions.")
-  :visibility   :authenticated
-  :feature      :email-restrict-recipients
-  :type         :keyword
-  :default      :all)
-
 (set! *warn-on-reflection* true)
 
 (when config/ee-available?
   (classloader/require 'metabase-enterprise.sandbox.api.util
                        'metabase-enterprise.advanced-permissions.common
                        'metabase-enterprise.advanced-permissions.models.permissions.group-manager))
 
+(defsetting user-visibility
+  (deferred-tru "Note: Sandboxed users will never see suggestions.")
```

Comment:
What is the link between user-visibility and suggestions?

Reply:
`user-visibility` is a setting that defines whether users should see other users in the dropdown when creating a subscription (aka "suggestions" I think)

But I'm not changing anything about the setting here, just moved it a few lines down since usually the `*warn-on-reflection*` and classloader stuff is at the top in most namespaces